dom0-updates/qubes-dom0-update

```diff
@@ -66,12 +66,14 @@ done
 # Prevent implicit update of template - this would override user changes -
 # but do allow explicit template upgrade, downgrade, reinstall
 if [ "$YUM_ACTION" == "reinstall" ] || [ "$YUM_ACTION" == "upgrade" ] || [ "$YUM_ACTION" == "upgrade-to" ] \
-|| [ "$YUM_ACTION" == "downgrade" ] && [[ "$PKGS" == *"qubes-template-"* ]]; then
+|| [ "$YUM_ACTION" == "downgrade" ] || [ "$YUM_ACTION" == "search" ] \
+|| [ "$YUM_ACTION" == "info" ] && [[ "$PKGS" == *"qubes-template-"* ]]; then
```

Comment:
It may be a better idea to handle info and search separately, instead of bundling it to upgrade etc. The way currently it is, is getting quite complex.
Something like another `elif` just before `else` at line 98/102

Reply:
But, without handling this here, the command will shutdown the template VM. Which is not necessary in case of `search` or `info` commands. 

@marmarek Do you think we should still shutdown the templates for those two commands? This is the only reason I added it that level.

Reply:
Wait, I am too sleepy.

Reply:
@marmarek You are correct, we can move this out to another elif block. Updating the PR.

Reply:
I mean instead of both of those modified places, not just the second one.